web-ui/controller/git_forge.ml

```diff
@@ -93,9 +106,16 @@ module Make (View : View) = struct
     Capability.with_ref (Client.CI.org ci org) @@ fun org_cap ->
     Capability.with_ref (Client.Org.repo org_cap repo) @@ fun repo_cap ->
     Capability.with_ref (Client.Repo.commit_of_hash repo_cap hash)
-    @@ fun commit ->
-    Client.Commit.jobs commit >>!= fun jobs ->
-    Client.Commit.refs commit >>!= fun refs ->
+    @@ fun commit_cap ->
+    Client.Commit.status commit_cap >>!!= fun status ->
+    Client.Commit.jobs commit_cap >>!= fun jobs ->
+    Client.Commit.refs commit_cap >>!= fun refs ->
+    let build_status : Client.State.t =
+      match status with
+      | `Failed -> Failed ""
+      | `Pending | `Not_started -> NotStarted
+      | `Passed -> Passed
```

Comment:
Shouldn't this live in `Client.State` as we might want to change it in the future?